packages/ui/src/kit/VInput/VInput.theme-default.scss

```diff
@@ -0,0 +1,29 @@
+.v-input.theme-default .v-input-reset-button {
+  line-height: 0;
+}
+.v-input.theme-default .v-input-reset-button svg {
```

Comment:
input does not know anything about svg

Reply:
you can use `<button>` tag with `<svg-atom>`

Reply:
In this specific case I can do without `svg` and line-height, with `flex` (required for auto alignment of svg inside).

Funny but fact, `svg-atom` selector using everywhere - in `v-button`, sidebar and navigation. These components does not know anything about svg.